lib/default-options.js

```diff
@@ -17,4 +17,5 @@ module.exports = {
   connectRetries: 2,
   downloadRetries: 2,
   attach: false,
+  maxDuration: 1800,
```

Comment:
but you're right - makes sense to default to `null` anyway rather than duplicate the Saucelabs detault here